Flask_app/views/auth.py

```diff
@@ -1,12 +1,19 @@
+from flask_login import logout_user
+from datetime import timedelta
 from flask import request ,jsonify
 from flask.views import MethodView
 from sqlalchemy.exc import IntegrityError
-
-from flask_app.models.models import UserRole, User
-from flask_app.services.auth import get_user_by_email,create_user
-
-
-
+from flask_app.models.models import User
```

Comment:
Import your models into models/__init__.py
In this way you can import it like a module